diff --git a/node_modules/react-native-password-strength-meter/src/index.js b/node_modules/react-native-password-strength-meter/src/index.js
index 7d47040..0eaf271 100644
--- a/node_modules/react-native-password-strength-meter/src/index.js
+++ b/node_modules/react-native-password-strength-meter/src/index.js
@@ -37,10 +37,11 @@ class PasswordInputComponent extends Component {
         levels = PASSWORD_INPUT.passwordProps.levels,
         minLength = PASSWORD_INPUT.passwordProps.minLength,
         scoreLimit = PASSWORD_INPUT.passwordProps.scoreLimit,
+        variations = PASSWORD_INPUT.passwordProps.variations,
       },
     } = this.props;
-    console.log({ levels, minLength, scoreLimit });
-    const score = scorePassword(password, minLength, scoreLimit);
+    // console.log({ levels, minLength, scoreLimit });
+    const score = scorePassword(password, minLength, scoreLimit, variations);
     const { label, labelColor, activeBarColor } = calculateLevel(score, levels);
     this.setState({
       password,
